app/lib/common/notifications/models.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/common/utils/utils.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart' as ffi;
+
+class NotificationBrief {
```

Comment:
Why not just compose it with ```freezed``` class? Makes it easier for handling states in ```StateNotifier``` and for immutability?

Reply:
By converting it into freezed, it automatically generates class constructor and ```copyWith``` methods with minimal code sample. If you think that won't benefit simplicity here, you can ignore it :)